in_toto/models/layout.py

```diff
@@ -256,13 +266,10 @@ def _validate_type(self):
 
   def _validate_threshold(self):
     """Private method to check that the threshold field is set to an int."""
-    try:
-      # int(self.threshold)
-      # FIXME: we don't have support for threshold yet
-      pass
-    except Exception as e:
+    if type(self.threshold) != int:
       raise securesystemslib.exceptions.FormatError(
-          "Invalid threshold value for this step. Exception: {}".format(e))
+          "Invalid threshold '{}' value, must be an int."
+          .format(self.threshold))
```

Comment:
Minor-issimo:
I'd put the comma (`,`) before `value`, or `value` right after `threshold`, e.g.: *"Invalid threshold '{}', value must be an int."* or *"Invalid threshold  value '{}', must be an int.* :)